backtrace_with_time/backtrace_with_time.py

```diff
@@ -0,0 +1,61 @@
+'''
+Adds a ubt command which adds basic block counts to frames within a backtrace.
+   Usage: ubt
+
+Contributors: Isa Smith, Toby Lloyd Davies
+Copyright (C) 2019 Undo Ltd
+'''
+
+
+from __future__ import absolute_import, print_function
+import gdb
+from undodb.debugger_extensions import udb
+
+
+class BacktraceWithTime(gdb.Command):
+    def __init__(self):
+        super(BacktraceWithTime, self).__init__('ubt', gdb.COMMAND_USER)
+
+    @staticmethod
+    def invoke(arg, from_tty):
+
```

Comment:
Not worth changing I think, but we tend to avoid blank lines after a function's first line.

Reply:
Done